packages/web-components/src/components/masthead/cloud/cloud-masthead.scss

```diff
@@ -95,7 +95,8 @@
     bottom: 0;
     width: 100%;
     height: carbon--rem(48px);
-    background: linear-gradient(rgba(255, 255, 255, 0.001), $ui-background); /* transparent keyword is broken in Safari */
+    background: linear-gradient(rgba(255, 255, 255, 0.001), $ui-background);
+    /* transparent keyword is broken in Safari */
```

Comment:
This appears to no longer be true, but is perhaps out of scope of this issue.

https://codepen.io/andy-blum/pen/oNqOVPO

Reply:
[Looks like there aren't a ton of instances of this](https://github.com/search?q=org%3Acarbon-design-system+transparent+keyword+is+broken+in+Safari&type=code), so maybe we should fix here

Reply:
cc @kennylam

Reply:
@andy-blum This was a solution that I've added to address the linear gradient transparency misbehaving only in Safari around _15 months ago_ when we first developed the Cloud Masthead. Since this is no longer an issue, we can certainly adjust it to `linear-gradient(to bottom, white, transparent);`.